### PR TITLE
interfaces: drop ErrUnknownSecurity

### PR DESCRIPTION
**Pull request by zyga**
This mammoth patch discards the ErrUnknownSecurity error code. The
motivation behind this is that there is an immense amount of boilerplate
code that is just there to say "I have nothing to do". With this change
in place it is sufficient to simply return a nil snippet.

Signed-off-by: Zygmunt Krynicki <zygmunt.krynicki@canonical.com>

### CODE REVIEW

File: interfaces/builtin/bluez.go [hunk 1]
```diff
@@ -210,10 +205,8 @@ func (iface *BluezInterface) ConnectedPlugSnippet(plug *interfaces.Plug, slot *i
 		return snippet, nil
 	case interfaces.SecuritySecComp:
 		return bluezConnectedPlugSecComp, nil
-	case interfaces.SecurityUDev, interfaces.SecurityDBus, interfaces.SecurityMount, interfaces.SecurityKMod:
-		return nil, nil
 	default:
-		return nil, interfaces.ErrUnknownSecurity
+		return nil, nil
```
Comment (niemeyer) [inline interfaces/builtin/bluez.go]: Can we please drop default in all those cases and move the return down?

**Comment by mvo5** [comment]
What not to like about `+90 −1,040` :-D

File: interfaces/builtin/bluez.go (hunk 1 above)
Comment (zyga) [inline interfaces/builtin/bluez.go]: Done

**Comment by zyga** [comment]
Updated as requested. Even more red!

**Pull request merged**